### ISSUE TITLE
404 in event detail

### ISSUE THREAD

**Issue by webian** [cdaecke/md_calendarize_frontend#1]
Hi,
after install and configure this extension, when I browse the event detail I get 404.
The event detail backs to work if I disable this extension or if I comment these lines:
https://github.com/cdaecke/md_calendarize_frontend/blob/948b95f935ac73b2411d3eb7efc8f6d552fa8c74/ext_localconf.php#L30-L34

**Comment by cdaecke** [comment]
Could you please paste the error message which you get?

Which version of TYPO3, ext:calendarize and ext:md_calendarize_frontend are you using?

Do you have other extensions installed, which interfere with ext:calendarize?

**Comment by webian** [comment]
> Could you please paste the error message which you get?

No TYPO3 or PHP errors, just a 404 browsing any event detail

> Which version of TYPO3, ext:calendarize and ext:md_calendarize_frontend are you using?

TYPO3 9.5.20
Calendarize 6.0.0
md_calendarize_frontend: 1.0.0
 
> Do you have other extensions installed, which interfere with ext:calendarize?

I don't think so.

Can you reproduce this? Else, don't worry, I will debug it because probably is something specific to my installation.

**Comment by cdaecke** [comment]
Hm, I can't reproduce this error in a fresh installation of TYPO3 9.5.19 nor with TYPO3 10.4.8

**Comment by cdaecke** [comment]
@webian Did you solve this problem?

**Comment by webian** [comment]
Not yet sorry. Too busy. 
But you can close this bug if you wish.

**Comment by cdaecke** [comment]
No, I'll keep it open, since I am also very interested in a solution :-)

If I can be of any help, please let me know.
